crates/router/src/core/payment_link/payment_link_initiate/payment_link.js

```diff
@@ -788,24 +817,21 @@ function handleCartView(paymentDetails) {
           item,
           paymentDetails,
           index >= MAX_ITEMS_VISIBLE_AFTER_COLLAPSE,
+          // @ts-ignore
           cartItemsNode
         );
       });
     }
-    if (cartItemsNode instanceof HTMLDivElement) {
```

Comment:
Why are we removing these checks?

Reply:
Having this checks in place causes the script to fail and not load the expected behaviour, you can check that the bug in show_more in our latest sandbox deployment.

Reply:
These checks are in place to avoid null pointer failures. I wouldn't recommend removing these as it **_might_** lead to complete failure of the JS script 

If there is a bug, it should be about how these conditions are written